microsoft-365/security/office-365-security/threat-explorer.md

```diff
@@ -74,9 +74,42 @@ You should also see additional details within the product. For example, the tota
 
 ![View In-product Info](../media/ProductInfo.png)
 
+## Extended capabilities in Threat Explorer
+
+### Top Targeted Users
+
+Today we expose the list of Top Targeted users in the Malware View for Emails (within the Top Malware Families section). We will be extending this view within Phish and All Email views as well, where you will be able to see the top 5 targeted Users along with no. of attempts for each user for corresponding view (e.g. for Phish view you will be able to see the number of Phish attempts ).
+You will also be able to export the list of Targeted users up to a limit of 3000 along with the no. of attempts for offline analysis for each email view. In addition to that, clicking on No. of attempts (e.g. 13 attempts below) would open a filtered view in Threat Explorer, so that you can look at more details across emails and threats for that user . 
```

Comment:
```suggestion
You will also be able to export the list of targeted users up to a limit of 3000 along with the number of attempts for offline analysis for each email view. In addition to that, selecting _No. of attempts_ (for example, 13 attempts below) would open a filtered view in Threat Explorer, so that you can look at more details across emails and threats for that user. 
```